api/lib/models/RSVP/RSVP-data-mapper-impl.ts

```diff
@@ -0,0 +1,222 @@
+import { Inject, Injectable } from 'injection-js';
+import { from } from 'rxjs';
+import { map } from 'rxjs/operators';
+import squel from 'squel';
+import { Stream } from 'ts-stream';
+import { UidType } from '../../JSCommon/common-types';
+import { HttpError } from '../../JSCommon/errors';
+import { AuthLevel } from '../../services/auth/auth-types';
+import { IAcl, IAclPerm } from '../../services/auth/RBAC/rbac-types';
+import { IDbResult } from '../../services/database';
+import { GenericDataMapper } from '../../services/database/svc/generic-data-mapper';
+import { MysqlUow } from '../../services/database/svc/mysql-uow.service';
+import { IUowOpts } from '../../services/database/svc/uow.service';
+import { Logger } from '../../services/logging/logging';
+import { IActiveHackathonDataMapper } from '../hackathon/active-hackathon';
+import { IRSVPDataMapper } from './index';
+import { RSVP } from './RSVP';
+
+@Injectable()
+export class RSVPDataMapperImpl extends GenericDataMapper
+    implements IAclPerm, IRSVPDataMapper {
```

Comment:
Can't find the IRSVPDataMapper interface definition. Doesn't seem to be added on this branch?

Reply:
Nvm I found it. Resolving.